src/development/tools/hot-reload.md

```diff
@@ -194,7 +194,7 @@ versus a hot restart.
 
 In Dart, [static fields are lazily initialized][const-new].
```

Comment:
The const-final portion of the language tour doesn't talk about this like the article did, so we should adjust this piece to link elsewhere in the tour. I've made this and one other comment to include those changes:

```suggestion
In Dart, [static fields are lazily initialized][static-variables].
```